driver-core/src/main/com/mongodb/internal/operation/ClientBulkWriteOperation.java

```diff
@@ -186,17 +187,17 @@ public String getCommandName() {
     @Override
     public MongoNamespace getNamespace() {
         // The bulkWrite command is executed on the "admin" database.
-        return new MongoNamespace("admin", COMMAND_COLLECTION_NAME);
+        return new MongoNamespace(ADMIN_DB_COMMAND_NAMESPACE.getDatabaseName(), COMMAND_COLLECTION_NAME);
```

Comment:
This could be simplified
```suggestion
        return ADMIN_DB_COMMAND_NAMESPACE;
```